backend/conferences/querysets.py

```diff
@@ -8,3 +11,7 @@ def for_conference(self, conference):
 
     def for_conference_code(self, conference):
         return self.filter(**{f"{self.conference_lookup_field}__code": conference})
+
+
+class ConferenceVoucherQuerySet(ConferenceQuerySetMixin, models.QuerySet):
```

Comment:
Why crating a custom class ` ConferenceVoucherQuerySet` if does not add anything? Can't you just use `ConferenceQuerySetMixin` then?

Reply:
ConferenceQuerySetMixin contains only the logic of the mixin, it isn't a `models.QuerySet`